### PR TITLE
Fixes #28252 - rules for foreman cockpit session

### PR DESCRIPTION
**Pull request by lzap**
Fixes a denial when switching from Foreman to Cockpit. Tested with Satellite 6.7 snap 2, however this should work with Foreman 1.24 too.

https://bugzilla.redhat.com/show_bug.cgi?id=1770999#c4

### CODE REVIEW

File: foreman.fc
Comment (ekohl) [inline foreman.fc:57]: This makes me wonder if we should move the file in packaging rather than making a symlink to it in `/usr/sbin`.

File: foreman.te
Comment (ekohl) [inline foreman.te]: The placing here implies this is REX but it's core Foreman functionality.

Comment (lzap) [reply foreman.te]: Good catch, I've moved this block accidentally actually.

File: foreman.fc
Comment (lzap) [reply foreman.fc:57]: I don't mind but this is a typical pattern we use across many packages.

**Comment by lzap** [comment]
Rebased, if you can smash the merge button we need this to unblock QA automation. This is tested end-to-end, I guarantee it works fine don't test it.

File: foreman.fc
Comment (ekohl) [reply foreman.fc:57]: Given this reflects the current package, I'm fine with merging it.

**Review by ekohl — APPROVED** [review]

**Comment by lzap** [comment]
Thank you, CP 1.24: https://github.com/theforeman/foreman-selinux/pull/96

**Pull request merged**
